docs/SmartStore.md

```diff
@@ -265,3 +265,7 @@ remote.s3.encryption = sse-s3
 ```
 2. Apply the CR with the necessary & supported Smartstore and Index related configs
 3. Install the App created using the [currently supported methods](https://splunk.github.io/splunk-operator/Examples.html#installing-splunk-apps) (*Note: This can be combined with the previous step*)
+
+## Special Internal Indexes
+
+The `_cluster` app sets the `repFactor` to `0` for internal indexes such as `_metrics`, `_introspection`, `_telemetry`, `_metrics_rollup`, `_configtracker`. If replication of these indexes are desired please use the method specified in [Additional Configuration section](#additional-configuration) to set the `repFactor` to `auto`
```

Comment:
How about something like:
For Indexer Cluster, all smartstore-enabled indexes must have a repFactor set to "auto." However, by default, the Cluster Manager's  _cluster app(by default) sets the repFactor to 0 for internal indexes like _metrics, _introspection, _telemetry, _metrics_rollup, and _configtracker. If you want to replicate these indexes, please follow the instructions in the Additional Configuration section to set the repFactor to "auto."